docs/cristin-proxy-internal-apidoc.yaml

```diff
@@ -0,0 +1,1074 @@
+openapi: 3.0.3
+info:
+  title: Internal NVA Cristin Proxy API
+  description: >
+    The NVA Cristin Proxy API.
+
+      This API provides transformed Cristin data stored in [Cristin](https://api.cristin.no/v2).
+
+      The data is reformatted as JSON-LD with a schema defined in the [search ontology](https://bibsysdev.github.io/src/search-ontology.ttl), the [Organization ontology](https://bibsysdev.github.io/src/organization-ontology.ttl) and the [project ontology](https://bibsysdev.github.io/src/publication-channel-ontology.ttl).
+
+      To understand the semantic layer, take a look at the @context URI returned by the API, the data is marked up according to this and may be interpreted directly to RDF using this.
+  termsOfService: http://nva.unit.no/terms/
+  contact:
+    email: nva@unit.no
+  license:
+    name: MIT/NLOD 2.0
+    url: https://link.to.license
+  version: 1.0.0
+externalDocs:
+  description: Find out more about NVA
+  url: https://nva.unit.no
+servers:
+  - url: https://api.nva.unit.no/cristin
+  - url: https://api.dev.nva.aws.unit.no/cristin
+  - url: https://api.sandbox.nva.aws.unit.no/cristin
+  - url: https://api.test.nva.aws.unit.no/cristin
+paths:
+  /person/identityNumber:
+    post:
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaCristinFetchPersonUsingNinFunction.Arn}/invocations
+        httpMethod: POST
+        type: "AWS_PROXY"
+      tags:
+        - Person
+      summary: Query person by national identity number
+      description: Returns a single person matching nin query parameter
+      operationId: GetPersonByNIN
+      security:
+        - CognitoUserPool: ['https://api.nva.unit.no/scopes/backend']
```

Comment:
So this is where the magic happens? Have you tested it locally? Should you write karate tests for it? Do you know if it works?